app/services/persoon-api.js

```diff
@@ -57,4 +57,22 @@ export default class PersoonApiService extends Service {
 
     await timeout(RESOURCE_CACHE_TIMEOUT);
   }
+
+  async putPersonInRightGraph(persoonId, bestuursorgaanID) {
+    const response = await fetch(
+      `${API.MANDATARIS_SERVICE}/personen/${persoonId}/put-person-in-right-graph/${bestuursorgaanID}`,
+      {
+        method: 'POST',
+      }
+    );
+    const jsonReponse = await response.json();
+
+    if (response.status !== STATUS_CODE.OK) {
+      console.error(jsonReponse.message);
+      throw {
```

Comment:
I see this is done in other places too but i believe it's actually not good practice and you should throw instances of an error object. the benefit is that you get a stack trace if it ends up with the user (or in our glitchtip environment).
don't necessarily do it for the existing cases, feel free to make a ticket for them